### ISSUE TITLE
Deprecation Warnings with jasmine-core 3.3.0 / karma 3.1.1

### ISSUE THREAD

**Issue by mnahkies** [dfederm/karma-jasmine-html-reporter#16]
Running tests with jasmine-core 3.3.0 and karma 3.1.1 gives:

```
ERROR: 'DEPRECATION:', 'Getting stoppingOnSpecFailure directly from Env is deprecated, please check the failFast option from `configuration`'
ERROR: 'DEPRECATION:', 'Getting stoppingOnSpecFailure directly from Env is deprecated, please check the failFast option from `configuration`'
ERROR: 'DEPRECATION:', 'Getting throwingExpectationFailures directly from Env is deprecated, please check the oneFailurePerSpec option from `configuration`'
ERROR: 'DEPRECATION:', 'Getting throwingExpectationFailures directly from Env is deprecated, please check the oneFailurePerSpec option from `configuration`'
ERROR: 'DEPRECATION:', 'Getting randomTests directly from Env is deprecated, please check the random option from `configuration`'
ERROR: 'DEPRECATION:', 'Getting randomTests directly from Env is deprecated, please check the random option from `configuration`'
ERROR: 'DEPRECATION:', 'Getting hidingDisabled directly from Env is deprecated, please check the hideDisabled option from `configuration`'
ERROR: 'DEPRECATION:', 'Getting hidingDisabled directly from Env is deprecated, please check the hideDisabled option from `configuration`'
```

Which appears to be coming from usages like: https://github.com/dfederm/karma-jasmine-html-reporter/blob/master/src/lib/html.jasmine.reporter.js#L378

**Comment by dfederm** [comment]
Will take a look at this later today.

**Comment by dfederm** [comment]
Fixed in 1.4.0

**Comment by dfederm** [comment]
There is one more that I see:
```
ERROR: 'DEPRECATION:', 'Setting specFilter directly on Env is deprecated, please use the specFilter option in `configure`'
```

But that seems to be coming from karma-jasmine, not this package
